### PR TITLE
High performance challenge update

### PR DESCRIPTION
**Pull request by milasuperstar**
Corrected date formats and use of language.
@dcalvin

### CODE REVIEW

File: challenge-programs/high-performance-tidb-challenge.md [hunk 1]
```diff
@@ -232,32 +232,65 @@ In this period, your team will be working together with the mentor to diagnose p
...
+
+TiDB Cloud is managed and supported by the same team that has designed, tuned, and operated TiDB for about 1,000 companies across multiple mission-critical segments such as financial services, e-commerce platforms, SaaS solutions, and social network applications. Enterprise support is available 24/7 for TiDB Cloud users, so customers can rest assured that their data is safe and their services are always on. 
+For more information, please visit the [TiDB Cloud Website](https://en.pingcap.com/products/tidbcloud)
+
+***How the credit coupons will be distributed***
+
+1st place: credit coupons worth 6,000 USD
```
Comment (dcalvin) [inline challenge-programs/high-performance-tidb-challenge.md:288]:
```suggestion
1st place:  6,000 USD worth of TiDB Cloud credits
```

File: challenge-programs/high-performance-tidb-challenge.md [hunk 2]
```diff
@@ -232,32 +232,65 @@ In this period, your team will be working together with the mentor to diagnose p
...
+
+***How to apply***
+
+Send an email to zhangyurui@pingcap.com with the subject formatted below.
+Subject: Your team name + application of server resource
+Make sure your Email address is valid as the server information will be sent by email.
+
+**TiDB Cloud credit coupons**
+
+To thank the contribution of participating teams, the organizing committee has prepared TiDB Cloud credit coupons worth 70,000 USD!
```
Comment (dcalvin) [inline challenge-programs/high-performance-tidb-challenge.md:279]:
```suggestion
To thank the contribution of participating teams, the organizing committee has prepared 70,000 USD worth of TiDB Cloud credit coupons!
```

**Review by dcalvin — DISMISSED** [review]
LGTM

**Comment by dcalvin** [comment]
Please also resolve the conflicts.

**Comment by milasuperstar** [comment]
> Please also resolve the conflicts.

Resolved.

**Comment by dcalvin** [comment]
@AndreMouche

**Review by AndreMouche — APPROVED** [review]

**Pull request merged**